src/main/kotlin/br/com/zup/op/events/exception/custom/HandlerAdvice.kt

```diff
@@ -0,0 +1,21 @@
+package br.com.zup.op.events.exception.custom
```

Comment:
remove this package `exception.custom` and move this class to `infra.exception` package